Tests/PostmarkClientTests/MailClientTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+import Mail
+import PostmarkClient
+import HTTP
+import Transport
+import Vapor
+
+class MailClientTests: XCTestCase {
+    static let allTests = [
+        ("testSendEmail", testSendEmail),
+    ]
+    
+    let testServerToken = "POSTMARK_API_TEST"
+    
+    func testSendEmail() throws {
+        let drop = Droplet()
+        let mailer = try PostmarkClient.MailClient(clientProtocol: drop.client.self, serverToken: testServerToken)
+        let email = Email(from: "from@email.com",
+                          to: "to1@email.com", "to2@email.com",
+                          subject: "Email Subject",
+                          body: "Hello Email")
+        try mailer.send(email)
+        
+        XCTAssertNotNil(mailer.result)
+        XCTAssertEqual(mailer.result!.status, Status.ok)
```

Comment:
Once the `send(_:)` method throws on any error, it will be as simple as asserting no error was thrown.